api/Dockerfile

```diff
@@ -7,3 +7,7 @@ WORKDIR $WORKDIR
 ADD . $WORKDIR
 
 RUN pip install -r $WORKDIR/requirements/dev.txt
+
+CMD python manage.py runserver
```

Comment:
run server by default for each docker build, we need that for migrations and user creation and gunicorn run

Reply:
understand